src/components/Frequency/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { forwardRef } from "react";
+import useIsDeprecatedForm from "../../hooks/internal/useIsDeprecatedForm";
+import { ComponentWithStaticProperties } from "../../types";
+import { ComponentNames } from "../componentNames";
+import { defaultFrequencyProps } from "./defaultFrequencyProps";
+import { default as FrequencyInternal } from "./Frequency";
+import { default as FrequencyDeprecated } from "./Frequency__deprecated";
+import {
+  FrequencyProps,
+  setDeprecatedProps as shouldUseDeprecatedInput,
+} from "./types";
+
+const { COLORS, DEFAULT_PROPS } = FrequencyDeprecated;
+
+interface StaticProperties {
+  displayName: string;
+  TYPES: string[];
+  VARIANTS: string[];
+  COLORS: string[];
+  DEFAULT_PROPS: typeof FrequencyDeprecated.DEFAULT_PROPS;
+}
+
+// can't be an arrow function due to: https://github.com/yannickcr/eslint-plugin-react/issues/2269
+const Frequency = forwardRef(function InputComponent(
```

Comment:
how about pulling `function InputComponent` out as FrequencyWithRef 
```
const FrequencyWithRef = (props: FrequencyProps, ref) => ....

const Frequency = forwardRef(FrequencyWithRef)
```

Reply:
I didn't change any of the code from the old frequency. It was like that to begin with